package.json

```diff
@@ -5,7 +5,7 @@
   "repository": "git@github.com:ministryofjustice/hmpps-education-and-work-plan-ui.git",
   "license": "MIT",
   "scripts": {
-    "prepare": "husky install",
+    "prepare": "husky",
```

Comment:
It appears `husky install` is to be deprecated in favour of `husky`,
https://github.com/typicode/husky/releases/tag/v9.0.1